docs/get-started/_index.md

```diff
@@ -1,36 +1,146 @@
 ---
-title: "Get Started"
-linkTitle: "Get Started"
+title: "Learn about Viam"
+linkTitle: "Learn about Viam"
 weight: 5
-type: "docs"
+description: "Viam is a complete software platform for smart machines which provides modular components and services for vision, motion, SLAM, ML, and data management."
 no_list: true
+type: docs
+aliases:
+  - "/getting-started/"
+  - "/getting-started/high-level-overview"
+  - "/product-overviews/"
+  - "/viam/"
+  - "/viam/app.viam.com/"
+imageAlt: "/general/understand.png"
+images: ["/general/understand.png"]
+carouselscript: true
 ---
 
-Learn about the Viam platform by reading our Viam in 3 minutes guide or try it out for yourself:
+Viam is a software platform that makes it easy to combine and integrate hardware and software to build machines, connect them with the cloud, and make them smarter with machine learning.
 
-{{< cards >}}
-{{% card link="/get-started/viam/" %}}
-{{% card link="/get-started/try-viam/" %}}
-{{% card link="/get-started/quickstarts/control-motor/" %}}
-{{% card link="/get-started/quickstarts/collect-data/" %}}
-{{% card link="/get-started/quickstarts/detect-people/" %}}
-{{% card link="/get-started/quickstarts/write-code/" %}}
-{{% card link="/get-started/quickstarts/drive-rover/" %}}
-{{< /cards >}}
+<div>
+{{< imgproc src="/viam/viam.png" alt="Viam overview" resize="800x" class="aligncenter" >}}
+</div>
 
-If you're ready to try it on your own machine, start by installing `viam-server`.
-If you need any help along the way, join our community:
+Machines are everywhere, from small machines like IOT sensors, to home automation systems, to robotic systems, cars and boats, and even more complex enterprise systems.
+All these machines start by combining hardware that senses or actuates through software.
```

Comment:
This sentence could be interpreted in a few ways:
1. Hardware senses *through* software.
    I don't think that's the intention, since most people probably think of it as hardware senses, and software interprets.
2. Hardware senses or actuates. Software is used to  combine pieces of hardware.
       What does it mean to combine?

Consider reworking to convey intended meaning

Reply:
will just simplify to hardware and software